### ISSUE TITLE
Daily replicator uses reporting

### ISSUE THREAD

**Issue by SodPen1** [IAmPicard/StarTrekTimelinesSpreadsheet#204]
**Describe the bug**
Incorrectly reports the number of replicator uses available.

**To Reproduce**
N/A

**Expected behavior**
Correct reporting of replicator uses remaining

**Screenshots**
N/A

**Additional context**
The tool currently reports that I have 5 out of 5 daily replicator uses remaining.  I currently have a max of 4 uses daily.  
Tool version: **0.8.1**
Operating system: **win32 x64 (10.0.17134)**

**Comment by IAmPicard** [comment]
Are you part of a fleet?

Can you try in the game to do a 5th replicate and let me know what the error message says?

**Comment by SodPen1** [comment]
Oh wow, I just figured out what's going on.  Your tool is more correct than the game.  LOL

My fleet has unlocked one level of use, and I'm VIP level 12.  That's a total of 5 daily replicator uses (1 native + 1 fleet perk + 3 VIP perks) and is absolutely being reported accurately by your tool.  However(!) my actual game client has only been letting me use it four times a day and I didn't even realize it should have been five until now when I went to burn all five to get the error message for you and it stops me after four.    

I'll close this and go ahead and pop open a ticket in-game with DB support.  My apologies.
